src/Umbraco.Web/PublishedCache/NuCache/ContentNodeKit.cs

```diff
@@ -4,7 +4,7 @@
 namespace Umbraco.Web.PublishedCache.NuCache
 {
     // what's needed to actually build a content node
-    internal struct ContentNodeKit
+    public struct ContentNodeKit
```

Comment:
Does this need to be public? If so we need to change this struct. It's internal for a reason because it simply uses fields. If this needs to be made into a real public API it needs to be changed.

Reply:
Yes, in order to implement IDataSource, ContentNode needs to be public. I'm open to ideas on how to refactor ContentNodeKit to make it less weird, not really sure how to improve.